tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,1251 @@
+    else:
+        intf1_indice = mg_facts['minigraph_ptf_indices'][intf1]
+        intf2_indice = mg_facts['minigraph_ptf_indices'][intf2]
+
+    asic.config_ip_intf(intf1, "10.10.1.2/28", "add")
+    asic.config_ip_intf(intf2, "10.10.1.20/28", "add")
+
+    yield intf1, intf2, intf1_indice, intf2_indice
+
+    asic.config_ip_intf(intf1, "10.10.1.2/28", "remove")
+    asic.config_ip_intf(intf2, "10.10.1.20/28", "remove")
+
+    if tbinfo['topo']['type'] != 't0':
+        if po1:
+            asic.config_portchannel_member(po1, intf1, "add")
+        if po2:
+            asic.config_portchannel_member(po2, intf2, "add")
+
+
+@pytest.fixture(scope='module')
+def ip_and_intf_info(config_facts, intfs_for_test, ptfhost, ptfadapter):
+    """
+    Calculate IP addresses and interface to use for test
+    """
+    ptf_ports_available_in_topo = ptfhost.host.options['variable_manager'].extra_vars.get("ifaces_map")
+
+    intf1_name, _, intf1_index, _, = intfs_for_test
+    ptf_intf_name = ptf_ports_available_in_topo[intf1_index]
+
+    # Calculate the IPv6 address to assign to the PTF port
+    vlan_addrs = list(list(config_facts['VLAN_INTERFACE'].items())[0][1].keys())
+    intf_ipv6_addr = None
+    intf_ipv4_addr = None
+
+    for addr in vlan_addrs:
+        try:
+            if type(ip_network(addr, strict=False)) is IPv6Network:
+                intf_ipv6_addr = ip_network(addr, strict=False)
+            elif type(ip_network(addr, strict=False)) is IPv4Network:
+                intf_ipv4_addr = ip_network(addr, strict=False)
+        except ValueError:
+            continue
+
+    # Increment address by 3 to offset it from the intf on which the address may be learned
+    if intf_ipv4_addr is not None:
+        ptf_intf_ipv4_addr = increment_ipv4_addr(intf_ipv4_addr.network_address, incr=3)
+        ptf_intf_ipv4_hosts = intf_ipv4_addr.hosts()
+    else:
+        ptf_intf_ipv4_addr = None
+        ptf_intf_ipv4_hosts = None
+
+    if intf_ipv6_addr is not None:
+        ptf_intf_ipv6_addr = increment_ipv6_addr(intf_ipv6_addr.network_address, incr=3)
+    else:
+        ptf_intf_ipv6_addr = None
+
+    logger.info("Using {}, {}, and PTF interface {}".format(ptf_intf_ipv4_addr, ptf_intf_ipv6_addr, ptf_intf_name))
+
+    return ptf_intf_ipv4_addr, ptf_intf_ipv4_hosts, ptf_intf_ipv6_addr, ptf_intf_name, intf1_index, intf1_name
+
+
+def generate_link_local_addr(mac):
+    """Generate ipv6 link local"""
+    parts = mac.split(":")
+    parts.insert(3, "ff")
+    parts.insert(4, "fe")
+    parts[0] = "{:x}".format(int(parts[0], 16) ^ 2)
+
+    ipv6Parts = []
+    for i in range(0, len(parts), 2):
+        ipv6Parts.append("".join(parts[i:i+2]))
+    ipv6 = "fe80::{}".format(":".join(ipv6Parts))
+    return ipv6
+
+# If/When NDP on ARPv6 becomes necessary and possible, will need to re-add in IPV6 parameterization option to this
+
+
+@pytest.fixture(params=['v4'])
```

Comment:
Seems v6 is missing here.

Reply:
See comment above method - since we cannot make rules for NDP with GCU yet, I have removed the param for v6, as any rule application to allow NDP would fail (pending rollout of the ICMP range changes), and therefore we don't want to return any ARP v6 (NDP) packets from this fixture.  I can remove all of the v6 logic and parameterization from the method, but had kept it in for ease of adding it back in if/when NDP is allowed with GCU rules

Reply:
Make sense. Thanks!